beacon-chain/sync/service.go

```diff
@@ -35,6 +35,9 @@ type Service struct {
 	crystallizedStateBuf        chan p2p.Message
 	announceActiveHashBuf       chan p2p.Message
 	activeStateBuf              chan p2p.Message
+	stateMapping                map[[32]byte]FinalizedBlock
```

Comment:
Use a better name? `stateMapping` sounds good generic.

Reply:
Im also not sure what this means

Reply:
sorry its a bit vague, maybe `StoredFinalizedBlocks` would be better ?